templates/python/custom-copilot-rag-azure-ai-search/env/.env.dev.user.tpl

```diff
@@ -0,0 +1,31 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
+
+# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
+SECRET_BOT_PASSWORD=
+{{#useOpenAI}}
+{{#openAIKey}}
+SECRET_OPENAI_API_KEY='{{{openAIKey}}}'
```

Comment:
Is the OpenAI key, endpoint collected from question model shared between dev and local environment? Or the information is only used for local debugging purpose?

Reply:
shared between dev and local envs